docs/en/pact-reference.md

```diff
@@ -388,31 +388,44 @@ as a unit.
 admin authorization schemes for modules and tables. Definition creates the keysets
 in the runtime environment and stores their definition in the global keyset database.
 
+#### Namespace declaration {#namespacedefinition}
+
+[Namespace](#namespaces) declarations provide a unique prefix for keysets and modules defined within the namespace scope. All smart contract definitions must occur within a namespaced scope, which is initiated by defining a namespace using [define-namespace](pact-functions.html#define-namespace), and entering the namespace scope by issuing the [namespace](pact-functions.html#namespace) command. The global (i.e. empty) namespace is reserved for Kadena code.
+
 #### Module declaration {#moduledeclaration}
 
-[Modules](#module) contain the API and data definitions for smart contracts. They are comprised of:
+[Modules](#module) contain an API implementation and data definitions for smart contracts.
```

Comment:
Can we leave lines alone that have no changes? Git isn't indicating what changed so I would rather you revert unchanged paras.